tests/python/opinfo_input_generators.py

```diff
@@ -1821,3 +1821,49 @@ def triu_error_generator(op: OpInfo, dtype: torch.dtype, requires_grad: bool = F
         yield SampleInput(
             make_arg(shape),
         ), RuntimeError, f"input tensor for triu must have 2 or more dims, but got {len(shape)} dims"
+
+
+def grouped_mm_input_generator(
+    op: OpInfo, dtype: torch.dtype, requires_grad: bool = False, **kwargs
+):
+    """
+    Generate valid test cases for grouped matrix multiplication.
+
+    Args:
+        op: OpInfo object for the bmm operation
+        dtype: Data type for test tensors
+        requires_grad: Whether tensors should require gradients
+    """
+    make_arg = partial(
+        make_tensor,
+        dtype=dtype,
+        device="cuda",
+        low=None,
+        high=None,
+        requires_grad=requires_grad,
+    )
+    make_index = partial(
+        torch.tensor, device="cuda", dtype=torch.int, requires_grad=False
+    )
+
+    # FIXME: enable test cases when switched to cublas fallback
```

Comment:
The cublas fallback is still in play? I thought you gave it up because it doesn't do fp8.

Reply:
nope that was an earlier comment that I haven't got around to update.